### ISSUE TITLE
[Bug] Shulk armor

### ISSUE THREAD

**Issue by DavidGodinho** [apace100/origins-fabric#376]
Using Origins 1.0.2 on a 1.17 server as well as some other [mods](https://pastebin.com/EkttZXSg).

My issue happens when putting on armor as a shulk race, tested with a full iron set vs just having no armor. Having no armor still reduces damage (from natural armor +8) whilst wearing armor made it take more damage than what the weapon would normally do (lost 4 hearts vs a 6.5 damage sword). The iron armor was crafted using the blacksmith class from Origins:Classes. When using the same armor with other races, the armor seemed to be working fine.

**Comment by apace100** [comment]
Hi. I sadly couldn't reproduce this. What I tried, everytime using an iron sword:
Hitting a shulk player without armor: 2 hearts of damage, then 2.5 hearts of damage
Hitting a shulk player with a regular iron chestplate: 1.5 hearts of damage twice, every third hit deals 2 hearts of damage
Hitting a shulk player with an iron chestplate created by a blacksmith: same as above, 1.5 hearts of damage twice, and every third hit 2 hearts of damage.
Hitting a shulk player with full regular iron armor: 4x 0.5 hearts of damage, then once 1 heart of damage (every fifth hit)
Hitting a shulk player with full blacksmith's iron armor:  same as with full regular iron armor

The blacksmith's bonus to armor toughness doesn't have an effect here because armor toughness specifically reduces damage taken from large hits, while the iron sword damage doesn't seem to be large enough to be affected by this.


However I'm seeing a mod called "attribute fix" in your list. Maybe that is the culprit?

**Comment by DavidGodinho** [comment]
I will test it out right now without attribute fix. I am using toro health to help get the numbers.

**Comment by DavidGodinho** [comment]
Yup, you are exactly right, it seems to be a conflict between your mod and attribute fix, I have decided to remove it for now.

**Comment by apace100** [comment]
I'll just reopen this as an incompatibility with the mod. :) Thanks for confirming!
